README.md

```diff
@@ -0,0 +1,50 @@
+# A command-line tool for managing [assets.ubuntu.com](https://github.com/canonical-web-and-design/assets.ubuntu.com)
+
+This is intended to be a simple command-line tool for uploading assets to an <https://assets.ubuntu.com>-like assets
+server.
+
+## Installation
+
+### Snap install
+
+On a [snap](https://snapcraft.io/)-enabled system, you can simply:
+
+``` bash
+sudo snap install upload-assets
+```
+
+### Pip install
+
+If you can't install the snap, you can install with pip.
+
+First [install python3 pip](http://stackoverflow.com/questions/6587507/how-to-install-pip-with-python-3), then:
+
+``` bash
+sudo pip3 install canonicalwebteam.upload-assets
+```
+
+## Usage
+
+You should now have access to the `upload-assets`command:
```

Comment:
missing a space after `upload-assets`.
not sure if that's what twine is complaining about?